knowledge-base/actions/damccorm/tag-ur-it/action-security.yml

```diff
@@ -0,0 +1,12 @@
+name: "tag-ur-it"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
```

Comment:
Please update input to `input: repo-token`, view [here](https://github.com/damccorm/tag-ur-it/blob/4689c0ef99e78276b2ff4851db5deec57bb3f615/action.yml#L8)